specs/unified_kernel_image.md

```diff
@@ -33,7 +33,10 @@ and UKIs follow the standard, so exact details will not be repeated here.
 
 UKIs can be generated via a [single, relatively simple `objcopy`
 invocation](https://www.freedesktop.org/software/systemd/man/systemd-stub.html#Assembling%20Kernel%20Images),
-that glues the listed components together, generating one PE binary that then can be signed for SecureBoot.
+that glues the listed components together, generating one PE binary
+that then can be signed for Secure Boot. Alternatively, a more bespoke
+tool to generate UKIs is [systemd's
+`ukify`](https://www.freedesktop.org/software/systemd/man/ukify.html).
```

Comment:
Hmm, so maybe drop this commit so that the one from #81 can be applied, which I think is better. (The text here still seems to recommend `objcopy` as the default approach, which is unlikely to get all the details right.)

Reply:
hm #81 is *this* PR?

Reply:
anyway, i dropped this commit now